src/content/web3/index.md

```diff
@@ -0,0 +1,148 @@
+---
+title: What is web3 and why is it important?
+description: An introduction to web3 - the next evolution of the internet - and why it matters.
+lang: en
+sidebar: true
+---
+
+# Introduction to web3 {#introduction}
+
+Centralization has helped onboard billions of people to the internet and created the stable, robust infrastructure on which it lives. At the same time, a handful of centralized entities have a stronghold on large swathes of the internet, unilaterally deciding what should and should not be allowed.
+
+Web3 is the answer to this dilemma. Instead of an internet monopolized by large technology companies, web3 embraces decentralization and is being built, operated, and owned by its users. Web3 will put power in the hands of individuals rather than corporations.
```

Comment:
We switched tense here.
```suggestion
Web3 is the answer to this dilemma. Instead of an internet monopolized by large technology companies, web3 embraces decentralization and is being built, operated, and owned by its users. Web3 puts power in the hands of individuals rather than corporations.
```